r/tests/testthat.R

```diff
@@ -20,6 +20,8 @@ library(arrow)
 library(tibble)
 
 if (identical(tolower(Sys.getenv("ARROW_R_DEV", "false")), "true")) {
+  arrow::arrow_info()
+
```

Comment:
This might be helpful in debugging. We can remove it before we merge this, though it should only impact our CI builds and maybe it's good/fine to keep it around?